python-packages/order_utils/src/zero_ex/json_schemas/__init__.py

```diff
@@ -30,7 +31,8 @@ def resolve_from_url(self, url: str) -> str:
 
         :param url: a string representing the URL of the JSON schema to fetch.
         :returns: a string representing the deserialized JSON schema
-        :raises jsonschema.ValidationError: when the resource associated with `url` does not exist.
+        :raises jsonschema.ValidationError: when the resource associated with
+                                            `url` does not exist.
```

Comment:
Reduce indent to be 4 spaces more than `:raises` on the line above.  Anal, yes, but this would make it consistent with the rest of the code.

Reply:
absolutely! it's important to keep the style consistent. Will make the change